internal/configuration/schema/authentication.go

```diff
@@ -35,6 +35,7 @@ type PasswordConfiguration struct {
 // AuthenticationBackendConfiguration represents the configuration related to the authentication backend.
 type AuthenticationBackendConfiguration struct {
 	DisableResetPassword bool                                    `mapstructure:"disable_reset_password"`
+	DisableDelayAuth     bool                                    `mapstructure:"disable_delay_auth"`
```

Comment:
As I said, I think we don't want to leave admins with a gun and let them shoot in their feet.